ticketer/recordlocator/views.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from django.shortcuts import render
+from django.http import HttpResponse
+
+from recordlocator import generator
+
+def record_locators(request):
+    """ View that generates at least one record locator, generating 'n' if
+    specified through a query parameter. """
+
+    num_locators = int(request.GET.get('n', 1))
```

Comment:
You know, it's funny but I was thinking that, just as I submitted this pull request. What do you think a decent max is? I can't imagine needing more than a 100 at a time?